modules/installation-configuration-parameters.adoc

```diff
@@ -1594,23 +1606,75 @@ Additional VMware vSphere configuration parameters are described in the followin
 |====
 |Parameter|Description|Values
 
-|`platform.vsphere.vCenter`
-|The fully-qualified hostname or IP address of the vCenter server.
+|`platform.vsphere.apiVIPs`
+|Virtual IP (VIP) addresses that you configured for control plane API access.
+a|Multiple IP addresses
+
+|`platform.vsphere.diskType`
+|Optional. The disk provisioning method. This value defaults to the vSphere default storage policy if not set.
+|Valid values are `thin`, `thick`, or `eagerZeroedThick`.
+
+|`platform.vsphere.failureDomains`
+|Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
 |String
 
-|`platform.vsphere.username`
-|The user name to use to connect to the vCenter instance with. This user must have at least
-the roles and privileges that are required for
-link:https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/vcp-roles.md[static or dynamic persistent volume provisioning]
-in vSphere.
+|`platform.vsphere.failureDomains.topology.networks`
+|Lists any network in the vCenter instance that contains the virtual IP addresses and DNS records that you configured.
 |String
 
-|`platform.vsphere.password`
-|The password for the vCenter user name.
+|`platform.vsphere.failureDomains.region`
+|You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
+|String
+
+|`platform.vsphere.failureDomains.zone`
+|You define a zone by using a tag from the `openshift-zone` tag category. The tag must be attached to the vCenter datacenter.
+|`String`
+
+|`platform.vsphere.ingressVIPs`
+|Virtual IP (VIP) addresses that you configured for cluster Ingress.
+|Multiple IP addresses
+
+|`platform.vsphere`
+| Describes your account on the cloud platform that hosts your cluster. You can use the parameter to customize the platform. When providing additional configuration settings for compute and control plane machines in the machine pool, the parameter is optional. You can only specify one vCenter server for your {product-title} cluster.
+|String
+
+|`platform.vsphere.vcenters`
+|Lists any fully-qualified hostname or IP address of a vCenter server.
+|String
+
+|`platform.vsphere.vcenters.datacenters`
+|Lists and defines the datacenters where {product-title} virtual machines (VMs) operate. The list of datacenters must match the list of datacenters specified in the `failureDomains` field.
+|String
+|====
+
+
+[id="deprecated-parameters-vsphere_{context}"]
+== Deprecated VMware vSphere configuration parameters
+
+In {product-title} 4.13, the following vSphere configuration parameters are deprecated. You can continue to use these parameters, but the installation program does not automatically specify these parameters in the `install-config.yaml` file.
```

Comment:
```suggestion
In {product-title} {product-version}, the following vSphere configuration parameters are deprecated. You can continue to use these parameters, but the installation program does not automatically specify these parameters in the `install-config.yaml` file.
```
Should fix this since this PR is for 4.12